RELEASE_NOTES.rst

```diff
@@ -21,6 +21,276 @@
 
 .. towncrier release notes start
 
+Airflow 2.9.0 (2024-03-28)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+
+Following Listener API methods are considered stable and can be used for production system (were experimental feature in older Airflow versions) (#36376):
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Lifecycle events:
+
+- ``on_starting``
+- ``before_stopping``
+
+DagRun State Change Events:
+
+- ``on_dag_run_running``
+- ``on_dag_run_success``
+- ``on_dag_run_failed``
+
+TaskInstance State Change Events:
+
+- ``on_task_instance_running``
+- ``on_task_instance_success``
+- ``on_task_instance_failed``
+
+Support for Microsoft SQL-Server for Airflow Meta Database has been removed (#36514)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+After `discussion <https://lists.apache.org/thread/r06j306hldg03g2my1pd4nyjxg78b3h4>`__
+and a `voting process <https://lists.apache.org/thread/pgcgmhf6560k8jbsmz8nlyoxosvltph2>`__,
+the Airflow's PMC and Committers have reached a resolution to no longer maintain MsSQL as a supported Database Backend.
+
+As of Airflow 2.9.0 support of MsSQL has been removed for Airflow Database Backend.
+
+A migration script which can help migrating the database *before* upgrading to Airflow 2.9.0 is available in
+`airflow-mssql-migration repo on Github <https://github.com/apache/airflow-mssql-migration>`_.
+Note that the migration script is provided without support and warranty.
+
+This does not affect the existing provider packages (operators and hooks), DAGs can still access and process data from MsSQL.
+
+Dataset URIs are now validated on input (#37005)
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+Datasets must use a URI that conform to rules laid down in AIP-60, and the value
+will be automatically normalized when the DAG file is parsed. See
+`documentation on Datasets <https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/datasets.html>`_ for
+a more detailed description on the rules.
+
+You may need to change your Dataset identifiers if they look like a URI, but are
+used in a less mainstream way, such as relying on the URI's auth section, or
+have a case-sensitive protocol name.
+
+To use the API delete methods of ``queuedEvent`` endpoints,  apache-airflow-providers-fab version 1.1.0 or higher is required. This version adds permission to delete a dataset. (#37176)
```

Comment:
1.1.0 ? So we should bump the version of FAB to 1.1.0 in main and release 1.1.0b0 then ?

Reply:
(so far we have 1.0.2)

Reply:
That relates to what I mentioned in https://github.com/apache/airflow/pull/38325#discussion_r1532552971
When we make the release there should be only 1 version of fab good to use. We don't need to hussle users with different versions that were good only for testing

Reply:
Yes. we will remove the others, so we can remove this line altogether. I think it's really a matter what version we agree to. If we remove 1.1.0 from here, we can stay with 1.0.2. I thin this is more reasonable approach to start with 1.0.* here. (but we have to have patchlevel versions going up because we cannot remove previously released versions).

Reply:
Summaizing: 
* let's remove this line
* let's stick with 1.0.2 version of fab being released next.